Homework-№1-from-07.09.2022/task7.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+// Task 7
+void sieve(int *arrayOfPrimeNumbers, int number)
+{
+    for (int i = 0; i <= number; ++i)
+    {
+        arrayOfPrimeNumbers[i] = true;
+    }
+    arrayOfPrimeNumbers[0] = false;
+    arrayOfPrimeNumbers[1] = false;
+    for (int i = 2; i <= number; ++i)
+    {
+        if (arrayOfPrimeNumbers[i]) {
+            for (int j = i * i; j <= number; j += i)
+            {
+                arrayOfPrimeNumbers[j] = false;
+            }
+        }
+    }
+}
+
+int main()
```

Comment:
```suggestion
void main()
```
Либо возвращай число в конце функции.